### DISCUSSION TITLE
Caution when merging - discussion before any more merges

### DISCUSSION THREAD

**Discussion by MarkW06071969** [MarkW06071969/Tafe202206#15] [General]
Hi all,

Sorry for the late notice, but we need to discuss potential issues for future merging. There are
no problems as yet but there could be if we are not all on the same page

Stephen is fine as he has done the first merge but we all need to be careful with subsequent merges
from Hamish and myself.

Namely the possibility of overwriting each other in the MainMenu.cs code.

Can we have a brief discussion about what to do in this regard going forth?

Maybe meet at say 1pm (if that is okay) in A1-14 (like Stephen and I did last week).

If Hamish can't be there, that's okay as he can be updated later.

These potential problems can be easily overcome if we are all aware  of what we need to do.

Thanks everyone,

Mark

**Comment by StephenHillenbrand** [comment]
I agree Mark, will be there to catchup. 
I think the only real problem will be the event handlers for each of the button on the MainMenu as you have suggested.

**Comment by 000961892** [comment]
I'll be there

**Comment by MarkW06071969** [comment]
Thanks Stephen, see you at 1pm 👍

Sent from Yahoo Mail on Android 
 
  On Thu, 1 Sept 2022 at 7:24 am, ***@***.***> wrote:   


I agree Mark, will be there to catchup.
I think the only real problem will be the event handlers for each of the button on the MainMenu as you have suggested.
